testsuite/features/support/env.rb

```diff
@@ -53,8 +53,8 @@
 Capybara.default_max_wait_time = ENV['CAPYBARA_TIMEOUT'] ? ENV['CAPYBARA_TIMEOUT'].to_i : 10
 DEFAULT_TIMEOUT = ENV['DEFAULT_TIMEOUT'] ? ENV['DEFAULT_TIMEOUT'].to_i : 250
 $is_cloud_provider = ENV['PROVIDER'].include? 'aws'
-$is_container_provider = ENV['PROVIDER'].include? 'podman'
-$is_container_server = %w[k3s podman].include? ENV.fetch('CONTAINER_RUNTIME', '')
+$is_gh_validation = ENV['PROVIDER'].include? 'podman'
```

Comment:
Just a doubt/nitpick

I see the env variable being set in the bash script under the  podman_runner directory, but why do we check for the PROVIDER to be  `podman` to know if we're running tests on GitHub ?
If it is an arbitrary value, maybe we can mention GH in it.

Reply:
Good point, Paolo. This needs to be clarified with @jordimassaguerpla, and we can improve this in a follow-up PR :wink: